components/apimgt/org.wso2.carbon.apimgt.common.analytics/src/main/java/org/wso2/carbon/apimgt/common/analytics/collectors/AnalyticsDataProvider.java

```diff
@@ -66,6 +66,8 @@ public interface AnalyticsDataProvider {
 
     String getUserAgentHeader();
 
+    default String getUserName(){ return null;};
```

Comment:
Please remove the return as this is an interface. No need to have access modifies as well

Reply:
Ok